homeassistant/components/zwave_js/config_flow.py

```diff
@@ -290,8 +293,9 @@ async def async_step_start_failed(
     async def _async_start_addon(self) -> None:
         """Start the Z-Wave JS add-on."""
         assert self.hass
+        assert self.addon_manager
         try:
-            await async_start_addon(self.hass, ADDON_SLUG)
+            await self.addon_manager.async_start_addon()
```

Comment:
Since it's a singleton, you don't need to store the addon manager on `self.addon_manager` and can just fetch it when you need it.